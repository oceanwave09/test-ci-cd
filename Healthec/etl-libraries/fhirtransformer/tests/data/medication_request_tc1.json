{
  "source_id": "12345689",
  "source_system": "http://www.bmc.nl/portal/prescriptions",
  "status": "active",
  "intent": "order",
  "medication_id": "e74abfd4-d039-451d-b949-690464e9a1ca",
  "patient_id": "69f946cb-a475-429d-ba9f-9ce0751a1904",
  "authored_date_time": "2015-01-15",
  "practitioner_id": "a10b4988-8e0d-4f9c-8af6-8a351f5afc5a",
  "reason_code": "92818009",
  "reason_display": "Chronic myeloid Leukemia (disorder)",
  "reason_system": "http://snomed.info/sct",
  "dosages": [
    {
      "sequence": "1",
      "text": "1 tablet every morning, 2 tablets everyday at 3:00"
    }
  ],
  "days_supply": "30",
  "quantity": "90"
}