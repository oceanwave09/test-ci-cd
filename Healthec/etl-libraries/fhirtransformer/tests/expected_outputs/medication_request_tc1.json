{
  "resourceType": "MedicationRequest",
  "identifier": [
    {
      "type": {
        "coding": [
          {
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "RI",
            "display": "Resource Identifier"
          }
        ]
      },
      "system": "http://www.bmc.nl/portal/prescriptions",
      "value": "12345689"
    }
  ],
  "status": "active",
  "intent": "order",
  "medicationReference": {
    "reference": "Medication/e74abfd4-d039-451d-b949-690464e9a1ca"
  },
  "subject": {
    "reference": "Patient/69f946cb-a475-429d-ba9f-9ce0751a1904"
  },
  "authoredOn": "2015-01-15",
  "requester": {
    "reference": "Practitioner/a10b4988-8e0d-4f9c-8af6-8a351f5afc5a"
  },
  "reasonCode": [
    {
      "coding": [
        {
          "system": "http://snomed.info/sct",
          "code": "92818009",
          "display": "Chronic myeloid Leukemia (disorder)"
        }
      ]
    }
  ],
  "dosageInstruction": [
    {
      "sequence": 1,
      "text": "1 tablet every morning, 2 tablets everyday at 3:00"
    }
  ],
  "dispenseRequest": {
    "quantity": {
      "value": 90
    },
    "expectedSupplyDuration": {
      "value": "30",
      "unit": "days",
      "system": "http://unitsofmeasure.org",
      "code": "d"
    }
  }
}